common/src/verify.rs

```diff
@@ -114,33 +114,40 @@ enum Phase {
     Block,
 }
 
-/// Verifies whether the given sequence of commits can be agenda subset of agenda finalized chain.
+/// Verifies whether the given sequence of next_block_commits can be agenda subset of agenda finalized chain.
 ///
 /// It may accept sequences that contain more than one `BlockHeader`.
 #[derive(Debug, Clone)]
 pub struct CommitSequenceVerifier {
     header: BlockHeader,
     phase: Phase,
     reserved_state: ReservedState,
-    commits: Vec<Commit>,
+    next_block_commits: Vec<Commit>,
+    total_commits: Vec<Commit>,
```

Comment:
what's this field for?

Reply:
I'm qurious too.

Reply:
See the use of `get_commits`

Reply:
Is it just for the integration test? I still don't get what this is for.

Reply:
> Is it just for the integration test? I still don't get what this is for.

Junha replied that it is probably useful in the future while creating the integration test